containers/scripts/ci_build_publish_docker_images.sh

```diff
@@ -114,4 +121,7 @@ fi
 if [ -z "${DOCKER_IMAGE_CONCLAVE_BUILD:-}" ] || [ "${DOCKER_IMAGE_CONCLAVE_BUILD}" == "1" ]; then
   buildAndPublishContainerIfItDoesNotExist $container_image_conclave_build buildContainerConclaveBuild
 fi
+if [ -z "${DOCKER_IMAGE_CONCLAVE_BUILD_INTEGRATION_TESTS:-}" ] || [ "${DOCKER_IMAGE_CONCLAVE_BUILD_INTEGRATION_TESTS}" == "1" ]; then
```

Comment:
These are env variables? Why is it useful to turn this off via this env variable?

Reply:
To speed up testing. It is nice to be able to disable building the containers that are not needed without having to change the scripts.